packages/eslint-plugin/src/rules/no-shorthand-prop.ts

```diff
@@ -0,0 +1,56 @@
+import { isPandaAttribute, isPandaProp, resolveLonghand } from '../utils/helpers'
+import { type Rule, createRule } from '../utils'
+
+export const RULE_NAME = 'no-shorthand-prop'
```

Comment:
any reason not to directly set config.shorthands to false ?

Reply:
Hmm...this actually makes sense.
cc @segunadebayo

Reply:
Agreed to keep this for Incremental adoption